public/docs/_examples/server-communication/dart/lib/wiki/wiki_smart_component.dart

```diff
@@ -0,0 +1,36 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'wikipedia_service.dart';
```

Comment:
Nit: relative imports come after all other kind of imports + separate groups by a newline.